### PR TITLE
Forbidden link

### PR DESCRIPTION
**Pull request by mvelosop**
## Summary

Comment on forbidden link

### CODE REVIEW

File: xml/System.Dynamic/DynamicObject.xml [hunk 1]
```diff
@@ -87,7 +87,8 @@ Dim sampleObject As Object = New SampleDynamicObject()
  [!code-vb[System.Dynamic.DynamicObject#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.dynamic.dynamicobject/vb/module1.vb#1)]  
   
  For more examples, see [Creating Wrappers with DynamicObject](https://go.microsoft.com/fwlink/?LinkId=169008) on the C# Frequently Asked Questions blog.
```
Comment (BillWagner) [inline xml/System.Dynamic/DynamicObject.xml]: This should change to https://devblogs.microsoft.com/csharpfaq/dynamic-in-c-4-0-creating-wrappers-with-dynamicobject/

Do you want to make this change @mvelosop ?

Comment (mvelosop) [reply xml/System.Dynamic/DynamicObject.xml]: Thanks @BillWagner, I'll do it 😊

Comment (mvelosop) [inline xml/System.Dynamic/DynamicObject.xml]:
```suggestion
 For more examples, see [Creating Wrappers with DynamicObject](https://devblogs.microsoft.com/csharpfaq/dynamic-in-c-4-0-creating-wrappers-with-dynamicobject/) on the C# Frequently Asked Questions blog.  
```

File: xml/System.Dynamic/DynamicObject.xml [hunk 2]
```diff
@@ -87,7 +87,8 @@ Dim sampleObject As Object = New SampleDynamicObject()
  [!code-vb[System.Dynamic.DynamicObject#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.dynamic.dynamicobject/vb/module1.vb#1)]  
   
  For more examples, see [Creating Wrappers with DynamicObject](https://go.microsoft.com/fwlink/?LinkId=169008) on the C# Frequently Asked Questions blog.  
-  
+ <!-- The link ABOVE returns 403 - Forbidden (I'm sorry to do this, there was no option to create an issue in the bottom of the page) -->
```
Comment (mvelosop) [inline xml/System.Dynamic/DynamicObject.xml]:
```suggestion
 
```

**Comment by mairaw** [comment]
It seems this is ready to merge. Do you want to take a look @BillWagner?

**Pull request merged**